entries/l/lhv.ee.json

```diff
@@ -1,13 +1,17 @@
 {
   "LHV": {
     "domain": "lhv.ee",
-    "url": "https://www.lhv.ee/",
-    "contact": {
-      "email": "info@lhv.ee",
-      "facebook": "LHVPank",
-      "twitter": "ellhaavee",
-      "language": "et"
-    },
+    "tfa": [
+      "custom-software",
+      "custom-hardware"
+    ],
+    "custom-software": [
+      "eIDAS (SmartID)"
+    ],
+    "custom-hardware": [
+      "eIDAS (ID-card, mobile-ID)",
+      "Security token"
```

Comment:
As only certificates validated by Estonian authorities are accepted it's not really true eIDAS is it? Because from reading https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=celex%3A32015D0296 it looks like interoperability is required for it to be eIDAS.

Reply:
It is true eIDAS.

Interoperability doesn't mean a business has to offer their services to everyone in the EU.

Reply:
True but saying they support eIDAS software when only one eIDAS software client is supported is a bit disingenuous. Leave it as SmartID for the software and ID-card, mobile-ID for the software instead.

Reply:
It's not. If someone required a specific manufacturer's attestation with Webautn, it'd still be Webauthn.

Reply:
So the user receives a x509 certificate to add to their browser?

> It's not disingenuous. In the same way, if someone required a specific manufacturer's attestation with Webauthn, it'd still be Webauthn.

If Google only allowed WebAuthn from Yubikey devices we'd label it as custom-hardware with a value of `Yubikey`.
For general consumers the behind the scenes technology is less important than the actual apps/devices that the can use. That's why the phrasing on our site is "Required hardware token: x" and not "Supported hardware standard: x".

Reply:
> So the user receives a x509 certificate to add to their browser?

Pretty much.

> If Google only allowed WebAuthn from Yubikey devices we'd label it as custom-hardware with a value of Yubikey.

And that's what it is currently, it is custom hardware with a value that describes the requirements. This doesn't mean Google doesn't follow Webauthn or that the devices can't be used elsewhere.